services/sputnik/SputnikHttpService/SputnikHttpService.ts

```diff
@@ -248,8 +248,8 @@ class SputnikHttpServiceClass {
       dao,
       userPermissions: {
         isCanCreateProposals:
-          isUserPermittedToCreateProposal(accountId, dao) ||
-          !!policyAffectsProposals.length,
+          isUserPermittedToCreateProposal(accountId, dao) &&
+          !policyAffectsProposals.length,
```

Comment:
u sure it should be && instead of || ?

Reply:
we want users to create proposals when 
1. User is authorized - we have an accountId
2. dao policy allows user to create new proposals
3. we have no active proposals that affects policy (chnageconfig proposals or change policy proposals)

so I suppose we have to use && here